pkg/orchestration/wiring/datadog/types.go

```diff
@@ -0,0 +1,57 @@
+package datadog
+
+import (
+	"github.com/atlassian/voyager"
+)
+
+type AlarmSpecType string
+type AlarmType string
+
+const (
+	Metric AlarmSpecType = "metric alert"
```

Comment:
is this the real enum value? having whitespace in enum value seems weird... or maybe it's not a enum?